<?xml version="1.0"?>
<launch>
  <!-- Remap cmd_vel before launching included move_base -->
  <group>
    <remap from="cmd_vel" to="/husky_velocity_controller/cmd_vel"/>

    <!-- Launch move_base with no static map -->
    <include file="$(find husky_navigation)/launch/move_base.launch">
      <arg name="no_static_map" value="true"/>
    </include>
  </group>

  <!-- Custom parameter overrides -->
  <rosparam command="load" file="$(find husky_waypoint_nav)/config/nav_overrides_working.yaml" ns="move_base"/>

</launch>
